gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.3-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.4.1-bin.zip
```

Comment:
intellij prefers the `-all` and will moan if you use the `-bin` version I think

Reply:
changed back to `all`. I don't think I've ever seen it moan apart from in plugin development